tests/docs/partials/too_large/test_too_many_messages.rst

```diff
@@ -0,0 +1,11 @@
+Scenario: An API consumer submitting a request with a request body containing 45,001 messages receives a 413 status code and response body containing an instance of that error
+===============================================================================================================================================================================
+
+| **Given** the API consumer provides a message body of 45,001 messages
+| **When** the request is submitted
+| **Then** the response is a 413 Request Entity Too Large
```

Comment:
```suggestion
| **Then** the response is a 413 Too Many Items
```

Reply:
Closing because we agreed to unify on Request Entity Too Large